packages/integration-tests/src/shared/templates.js

```diff
@@ -23,6 +23,7 @@ exports.todoApp = function (cmpName) {
 
 const COMPAT = `
     <script src="../../shared/downgrade.js"></script>
+    <script>delete window.customElements;</script>
```

Comment:
Same here, we may be masking some issue. This will get us past in test mode. But what about non-test mode?